dev/tests/functional/tests/MFTF/DevDocs/Helper/CustomHelper.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace MFTF\DevDocs\Helper;
+
+use Magento\FunctionalTestingFramework\Helper\Helper;
+
+class CustomHelper extends Helper
+{
+    /**
+     * Custom helper.
+     *
+     * @param string $url
+     * @param float  $test
```

Comment:
is there a reason why $test is set to float? I saw this on running the test, changing it to string fixed the problem -
` [TypeError] Argument 2 passed to MFTF\DevDocs\Helper\CustomHelper::goTo() must be of the type float, string given, called in /Users/sunnikri/workspace/magento2-functional-testing-framework/dev/tests/functional/tests/MFTF/_generated/default/DevDocsTestCest.php on line 54 `

Reply:
fixed